packages/core/src/queries/organization/relations.ts

```diff
@@ -196,6 +200,36 @@ export class RoleUserRelationQueries extends RelationQueries<
     `);
   }
 
+  /**
+   * Get the available resource scopes of a user in all organizations.
+   * if organizationId is provided, it will only search in that organization
```

Comment:
```suggestion
   * Get the available resource scopes of a user in all organizations.
   * If `organizationId` is provided, it will only search in that organization.
```